kalamine/tpl/base.ahk

```diff
@@ -61,12 +61,26 @@ SetTimer, ShowTrayTip, -1000  ; not working
 
 global DeadKey := ""
 
+; Check CapsLock status, upper the char if needed and send the char
+SendChar(unicode) {
```

Comment:
Sorry to nitpick again, but `unicode` is misleading here since this can either be a `U+xxxx` string or a single-byte character. Maybe `char` would do the job?